Simplenote/CrashLogging.swift

```diff
@@ -0,0 +1,125 @@
+import Foundation
+import AutomatticTracks
+
+/// This exists to bridge CrashLogging with Objective-C. Once the App Delegate is moved over to Swift,
+/// this shim can be deleted.
```

Comment:
Yeah, my thinking was you could move the caching settings into an extension without too much fuss since they're `static`. I thought about doing it and literally just having the shim bridge Obj-C to Swift, but that seemed like a lot of boilerplate. 

I'll update the comment to better describe exactly what should happen to that code.